rasa/cli/x.py

```diff
@@ -291,16 +304,76 @@ def rasa_x(args: argparse.Namespace):
         run_locally(args)
 
 
-def run_in_production(args: argparse.Namespace):
-    from rasa.cli.utils import print_success
-    from rasa.core.utils import AvailableEndpoints
+def _pull_runtime_config_from_server(
+    config_endpoint: Optional[Text],
+    attempts: int = 30,
+    wait_time_between_pulls: Union[int, float] = 0.5,
+    keys: Tuple[Text, ...] = ("endpoints", "credentials"),
```

Comment:
cool, never seen that before. 
- wait, isn't a tuple with unlimited items a `list`?

Reply:
no, the tuple is created with a fixed size. the type annotations just indicates that it's of variable length. but I agree, we can use a list instead

Reply:
actually we can't use a list as a default argument as it's mutable

Reply:
Then `Iterable` and put a tuple in there?